src/app/account-pet/account-pet.component.ts

```diff
@@ -0,0 +1,103 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, FormArray, Validators } from '@angular/forms';
+@Component({
+  selector: 'app-account-pet',
+  templateUrl: './account-pet.component.html',
+  styleUrls: ['./account-pet.component.css']
+})
+export class AccountPetComponent implements OnInit {
+
+  newPetForm: FormGroup;
```

Comment:
Will this component only be used for a new pet or will it also be used for updating an existing pet.  You can create a petsService class (even if it is fake for now) and then this component can be completed more.

Reply:
Yes, this form will also be used to update an existing pet. I will create a petsService class. Thanks!